.travis.yml

```diff
@@ -152,12 +150,14 @@ matrix:
         - ./support/ci/compile_libarchive.sh
         - ./support/ci/compile_zmq.sh
         - source ./support/ci/rust_env.sh
+        - cargo build --package habitat-builder-protocol
```

Comment:
A comment on why we need this would be useful.

Reply:
Added